rr/stream/stream_manager.py

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+
+#  Copyright (C) 2021 RidgeRun, LLC (http://www.ridgerun.com)
+#  Authors: Daniel Chaves <daniel.chaves@ridgerun.com>
+#           Marisol Zeledon <marisol.zeledon@ridgerun.com>
+
+from rr.ai.ai_manager import AIManagerOnNewImage
+from rr.gstreamer.media_manager import MediaManager
+
+model = "/opt/edge_ai_apps/models/detection/TFL-OD-200-ssd-mobV1-coco-mlperf-300x300/"
+disp_width = 2040
+disp_height = 1920
+
+
+class OnNewImage():
```

Comment:
Missing docstring

Reply:
Yes, docstring, will be handle in a PR for all the classes, that were recent or refactored; in order to avoid redocumenting close to the project release.